sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/ActivityExtensions.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using OpenTelemetry.Exporter.AzureMonitor.Models;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal static class ActivityExtensions
+    {
+        private static readonly string INVALID_SPAN_ID = default(ActivitySpanId).ToHexString();
+        private static readonly string INVALID_TRACE_ID = default(ActivityTraceId).ToHexString();
+
+        internal static string GetSpanId(this ActivitySpanId activitySpanId)
+        {
+            var spanId = activitySpanId.ToHexString();
+            if (!string.Equals(spanId, INVALID_SPAN_ID, StringComparison.Ordinal))
+            {
+                return spanId;
+            }
+
+            return null;
+        }
+
+        internal static string GetTraceId(this ActivityTraceId activityTraceId)
+        {
+            var traceId = activityTraceId.ToHexString();
+            if (!string.Equals(traceId, INVALID_TRACE_ID, StringComparison.Ordinal))
+            {
+                return traceId;
+            }
+
+            return null;
```

Comment:
Generally same comment as above, from an API design perspective unless String.Empty has a different meaning than null, you probably just want to default to string.empty, this way someone who's doing something like GetTraceId().ToLowerInvariant() won't get an unhappy surprise.